index.html

```diff
@@ -123,17 +132,19 @@ <h1 id="works-title">My Recent Works</h1>
     			<span class="tech">JavaScript</span>
     			<span class="tech">html</span>
     		</p>
-    		<button type="submit" class="project-button">See Project</button>
+    		<button type="button" class="project-button">See Project</button>
     	</div>
     </section>
+
     <section id="About-section">
     	   <h1 id="About-title">About me</h1>
 		   <p id="About-text">
 	    	   	I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded
 	    	   	, don’t hestiate to contact me.
     	   </p>
     	   <img id="About-img2" alt="About image 2" src="./images/aboutimg2.png">
-    	   <button type="submit" id="About-button">Get My Resume</button>
+    	   <button type="button" id="About-button">Get My Resume</button>
+    	   <div class="categories">
```

Comment:
- [ ] These section contents are well placed. But I found out that your design has a smaller space between the ending cards and the contact form section. I believe you can fix that by adding the spacing to match with the Figma design.
Also, take note of the input focus colours.

Your Design | 
:---------------:|
 ![e2](https://user-images.githubusercontent.com/39523312/149220678-d2d6be7f-6130-42fd-9314-153dc9a86517.png)

Figma Design |
:------------------:|
![e4](https://user-images.githubusercontent.com/39523312/149220683-475fae0f-4775-4e87-9c11-e53979f44e8f.png)